docs/_snippets/button-group-story.md

```diff
@@ -1,4 +1,4 @@
-```ts filename="ButtonGroup.stories.ts" renderer="angular" language="ts"
+````ts filename="ButtonGroup.stories.ts" renderer="angular" language="ts"
```

Comment:
Why does this have 4 ticks @jonniebigodes ?

Reply:
It shouldn't, probably introduced with a recent commit. @JReinhold can you adjust it.

Reply:
Fixed with the latest commit.